### PR TITLE
Deployment tooling update - bin/eb-env-exists

### PR DESCRIPTION
**Pull request by indigobravo**
The bin/eb-env-exists script has been updated to handle multi-region
deployments.

### CODE REVIEW

File: bin/eb-env-exists [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 usage () {
-    echo "Usage: $(basename "$0") <APP> <ENV>"
+    echo "Usage: $(basename "$0") <APP> <ENV> <REGION>"
     echo
     echo "Check if an Elastic Beanstalk environment already exists. Exits zero"
     echo "when the environment exists, and non-zero otherwise."
 }
 
-if [ "$#" -ne 2 ]; then
+if [ "$#" -ne 3 ]; then
```
Comment (indigobravo) [inline bin/eb-env-exists:10]: Increased mandatory parameters to 4 now we are passing in `$REGION`

File: bin/eb-env-exists [hunk 2]
```diff
@@ -16,10 +16,12 @@ set -eu
 
 APP=$1
 ENV=$2
+REGION=$3
```
Comment (indigobravo) [inline bin/eb-env-exists:19]: Set `$REGION` to $3

File: bin/eb-env-exists [hunk 3]
```diff
@@ -16,10 +16,12 @@ set -eu
...
 
 count=$(aws elasticbeanstalk describe-environments \
         --application-name "$APP" \
         --environment-names "${APP}-${ENV}" \
         --query 'length(Environments)' \
-        --output text)
+        --output text) \
+        --region "$REGION"
```
Comment (indigobravo) [inline bin/eb-env-exists:26]: Added `--region` parameter to aws command

Comment (robertknight) [reply bin/eb-env-exists:26]: The `--region` option has been added outside of the sub-shell `$(...)` here so it won't be passed to the `aws` command.

**Review by robertknight — CHANGES_REQUESTED** [review]
The change looks good, except for an error in how the `--region` argument is passed. Good to merge once that is resolved.

File: bin/eb-env-exists (hunk 3 above)
Comment (indigobravo) [reply bin/eb-env-exists]: Thanks for spotting this, a new update has been pushed.

**Review by robertknight — APPROVED** [review]

**Pull request merged**